Vagrantfile

```diff
@@ -73,6 +73,31 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     backend.vm.provision :shell, :inline => "facter --custom-dir=/vagrant/lib/facter sensu_backend"
   end
 
+  config.vm.define "sensu-backend-federated1", autostart: false  do |backend|
+    backend.vm.box = "centos/7"
+    backend.vm.hostname = 'sensu-backend-peer1.example.com'
+    backend.vm.network :private_network, ip: '192.168.52.30'
+    backend.vm.network :forwarded_port, guest: 2380, host: 2383, auto_correct: true
+    backend.vm.network :forwarded_port, guest: 3000, host: 3003, auto_correct: true
+    backend.vm.network :forwarded_port, guest: 8080, host: 8084, auto_correct: true
+    backend.vm.network :forwarded_port, guest: 8081, host: 8086, auto_correct: true
+    backend.vm.provision :shell, :path => "tests/provision_basic_el.sh"
+    backend.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-backend-federated-cluster.pp"
+    backend.vm.provision :shell, :inline => "sensuctl role list"
```

Comment:
a comment in here like if this works you should see X output at the end of the provisioning step would be helpful.

Reply:
I am documenting that in README with latest commit just pushed to support SSL